levels.js

```diff
@@ -177,13 +177,287 @@ var bonusLevelOne = {
 
 };
 
-// NOTE: THIS IS JUST A CODE SNIPPET.
-/* 
     var levelTwo = {
     underground: true, // NOTE: PLEASE ADD THIS BOOLEAN PROPERTY WHEN TESTING LEVEL 1-2
+
+var levelTwo = {
```

Comment:
Due to bad merge, this redundant line 183 will break the game. @BenjaminDeJager Please fix the issue, then we can merge.